### PR TITLE
https://issues.redhat.com/browse/ACM-12003

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

**Review by dislbenn — APPROVED** [review]
LGTM

File: install/install_disconnected.adoc [hunk 1]
```diff
@@ -147,13 +149,14 @@ You need the `metadata.name` field value for the annotation in the `Multicluster
...
 
+[source,bash]
 ----
 oc -n openshift-marketplace get packagemanifests 
 ----
 
 The list that is displayed should include entries showing that the following packages are supplied by 
 the catalog source for your mirror catalog:
-
+//D, should this also be updated to "management"?
```
Comment (dockerymick) [inline install/install_disconnected.adoc]: @dislbenn

Comment (dislbenn) [reply install/install_disconnected.adoc]: @dockerymick yes, we should update this here as well.

Comment (dockerymick) [reply install/install_disconnected.adoc]: Thanks for confirming! Updating now

**Review by jc-berger — APPROVED** [review]
/lgtm

**Comment by dockerymick** [comment]
@dislbenn and @jc-berger thank you!

**Pull request merged**